### PR TITLE
Create Blog Page with Post About Missed Fines

### LINKED ISSUE #75: Calculate Potential Fines Based on Number of Non-Submitting Buildings
Calculate the total amount of potential fines based on the number of non-compliant buildings in the data according to this ordinance: https://www.chicago.gov/content/dam/city/progs/env/EnergyBenchmark/SO2017-7060.pdf

1. Get latest Environment Sustainable dataset from the Chicago Data Portal [here](https://data.cityofchicago.org/Environment-Sustainable-Development/Chicago-Energy-Benchmarking/xq83-jr8c/explore/query/SELECT%0A%20%20%60data_year%60%2C%0A%20%20%60id%60%2C%0A%20%20%60property_name%60%2C%0A%20%20%60reporting_status%60%2C%0A%20%20%60address%60%2C%0A%20%20%60zip_code%60%2C%0A%20%20%60chicago_energy_rating%60%2C%0A%20%20%60exempt_from_chicago_energy_rating%60%2C%0A%20%20%60community_area%60%2C%0A%20%20%60primary_property_type%60%2C%0A%20%20%60gross_floor_area_buildings_sq_ft%60%2C%0A%20%20%60year_built%60%2C%0A%20%20%60of_buildings%60%2C%0A%20%20%60water_use_kgal%60%2C%0A%20%20%60energy_star_score%60%2C%0A%20%20%60electricity_use_kbtu%60%2C%0A%20%20%60natural_gas_use_kbtu%60%2C%0A%20%20%60district_steam_use_kbtu%60%2C%0A%20%20%60district_chilled_water_use_kbtu%60%2C%0A%20%20%60all_other_fuel_use_kbtu%60%2C%0A%20%20%60site_eui_kbtu_sq_ft%60%2C%0A%20%20%60source_eui_kbtu_sq_ft%60%2C%0A%20%20%60weather_normalized_site_eui_kbtu_sq_ft%60%2C%0A%20%20%60weather_normalized_source_eui_kbtu_sq_ft%60%2C%0A%20%20%60total_ghg_emissions_metric_tons_co2e%60%2C%0A%20%20%60ghg_intensity_kg_co2e_sq_ft%60%2C%0A%20%20%60latitude%60%2C%0A%20%20%60longitude%60%2C%0A%20%20%60location%60%2C%0A%20%20%60row_id%60%2C%0A%20%20%60%3A%40computed_region_43wa_7qmu%60%2C%0A%20%20%60%3A%40computed_region_vrxf_vc4k%60%2C%0A%20%20%60%3A%40computed_region_6mkv_f3dw%60%2C%0A%20%20%60%3A%40computed_region_bdys_3d7i%60%2C%0A%20%20%60%3A%40computed_region_awaf_s7ux%60%0AWHERE%0A%20%20%28%60data_year%60%20%3E%202018%29%0A%20%20AND%20caseless_one_of%28%60reporting_status%60%2C%20%22Not%20Submitted%22%29/page/filter). The export can be found here: 
[Chicago_Energy_Benchmarking_20240220_superset.csv](https://github.com/vkoves/electrify-chicago/files/14353461/Chicago_Energy_Benchmarking_20240220_superset.csv)
2. Find number of buildings not in compliance with city emission data requirements and their associated square footage.
3. Map buildings to fine levels based on building square footage.
4. Sum total fines based on those fines associated to the buildings 

Buildings that are non-compliant will show "Not Submitted" in the `Reporting Status` column of the data. The data in the link shown above should already be filtered.

### PR DESCRIPTION
**Pull request by vkoves**
# Description

Adds a new blog page with a new article about how the city missed out in $30 million in fines by not enforcing the energy benchmarking ordinance.

| Blog Page | Post Top |
| --- | --- |
| Screenshot from 2024-02-27 23-57-05 | Screenshot from 2024-02-27 23-57-26 |

Resolves #75

# Testing Instructions

Viewed the new blog page and post on desktop and mobile and ran AxeDevTools to confirm it has no accessibility issues.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by vkoves** [comment]
Suggestion in Slack from Soren:

> Parachuting in to nitpick that I think in several places where you use the word "buildings" alone to describe the party that is required to report, it may be better to use "building owners". It's like the "car" vs. "driver" distinction in traffic crash reporting.

File: src/pages/blog/MillionsInMissedFines.vue [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+      <img
+        src="/blog/millions-in-missed-fines/zoomed-non-reporting-count.webp"
+        class="blog-img"
+        width="400"
+        alt="A screenshot of text that reads 'Showing rows 1-100 of 3325'"
+      >
+
+      <p class="caption">
+        The full data portal screenshot, and a zoomed in view of the query result count (
```
Comment (derekeder) [inline src/pages/blog/MillionsInMissedFines.vue:104]: this line seems to be not rendering properly 

Screenshot 2024-02-28 at 4 30 14 PM

**Comment by derekeder** [comment]
this is great - hopefully it will turn a few heads. do you plan to pitch to a publication? block club, trib or sun-times might be interested.

a few notes:

- might be worth updating the home page to feature this blog post if you want more people to find/read it
- shorter version of the headline reads a little better: "City Of Chicago Failed to Collect $30 Million In Potential Fines from Building Benchmarking Ordinance"
- add a way to contact you for follow-up information

**Comment by vkoves** [comment]
@derekeder - great suggestions, I've added my external contact email to the post and added a new panel to the homepage:

Screenshot from 2024-02-28 17-00-07

Thoughts?

**Comment by derekeder** [comment]
nice. i think the two alerts are competing with each other with color a bit. might look better if they were stacked vertically instead of horizontal too.

**Comment by vkoves** [comment]
@derekeder - I did that before, but then you have to scroll down more on desktop view the actual table. I could make the legislative update background white, perhaps?

**Comment by derekeder** [comment]
yah white background would help i think

**Comment by vkoves** [comment]
Merging this in since folks have looked at it and it's not changing data

**Pull request merged**